pkg/db/tx.go

```diff
@@ -121,6 +121,9 @@ func (d *dbTx) Exec(query string, args ...interface{}) (pgconn.CommandTag, error
 		"query": queryToString(query),
 		"took":  time.Since(start),
 	})
+	if IsUniqueViolation(err) {
```

Comment:
This breaks `pkg/auth/service.go` - there are several places that check for `db.IsUniqueViolation(..)` - Having a good error value is indeed better but please fix the auth service to check for this error instead..
Better yet, make this check unexported to make sure no-one else is relying on it.

Reply:
Done.